scripts/codegen/index.js

```diff
@@ -0,0 +1,263 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
Shouldn't there be some tests for all this?

If we were to update dependencies we wouldn't know its failing before someone tries to generate updated types.

Reply:
Agree.